usefulUtilities/avatarEntityRemover/README.md

```diff
@@ -0,0 +1,35 @@
+# Avatar Entity Remover
+When this script is attached to an entity as an entity script, the entity serves as an "Avatar Entity Remover", useful for removing avatar entities in a domain.
+
+## Features
+- Avatar Entity Remover will prevent new avatar entities from being added by a user when the script is running.
+- Avatar Entity Remover will remove existing avatar entities from an avatar when they load the script.
+- If a user of a locked avatar entity loads the script, they will be kicked to a user-configurable domain.
+- Using a configuration switch, you can enable a feature that will restore all removed avatar entities when the Avatar Entity Remover script is unloaded.
+
+## Exclusions
+- You cannot whitelist avatar entity model domains - with that feature, users could simply enlarge approved wearables and use them for griefing purposes.
+
+## Setup
+1. Add an entity to your domain.
+    - A giant Zone entity would work perfectly. A good place for this entity is centered around your domain's content. Its dimensions should be large enough to encompass the domain's content. You want to ensure that all visitors to your domain load this entity.
+    - Note that anyone who can modify the `userData` of this entity will be able to control this script's configurable settings!
+2. Add the below `userData` object to the attached entity's `userData`
+    1. Set the `enableAvatarEntityRestore` `bool`.
+    2. Set the `kickDomain` value in the `userData`.
+    3. Set the `enableCollisionlessAvatarEntities` `bool`.
+3. Add the `avatarEntityRemover.js` script to the entity
+
+Here's the object to add to the entity's `userData`:
+```
+{
+    "enableAvatarEntityRestore": <`true` if you want removed avatar entities to be restored when the script is unloaded; `false` otherwise>,

```

Comment:
Not a big one, saw the markdown looked like this on the true.  
![image](https://user-images.githubusercontent.com/11789978/54724668-5b5bdc80-4b29-11e9-866f-24c5cb9aae4e.png)  Mentioning just in case that wasn't intended.

Reply:
yup, this is intended!